pyscript.core/types/3rd-party/codemirror.d.ts

```diff
@@ -1 +1 @@
-export * from "codemirror";
+export {};
```

Comment:
I think you need to update your TS locally ... these changes (this and others) have nothing to do with this MR so please revert these or update your global or local TS (npm i should do).